content/Software and Downloads/Installation/Add-a-library-to-Arduino-IDE.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Add a library to Arduino IDE"
+---
+
+Libraries are a collection of code that provide extra functionality for use in sketches, for example when working with hardware or manipulating data.
+
+Learn how to add a library to Arduino IDE.
+
+---
+
+## Using the Library manager
+
+1. In the menu bar, go to `Tools > Manage Libraries...` (In IDE 2, you can also click on the Library Manager icon on the left).
+
+2. Search for the library you want to install.
+
+3. Find the library in the search results.
+
+4. Click on the INSTALL button.
+
+   ![IDE 1 Library Manager with search box and Install button highlighted](img/LibraryManager_InstallingLibrary_IDE1.png)
+
+   ![IDE 2 Library Manager Icon, search box, and Install button highlighted](img/LibraryManager_InstallingLibrary_IDE2.png)
+
+5. Wait for the installation to complete.
+
+---
+
+## Importing a .zip Library
+
+1. In the menu bar, go to `Sketch > Include Library > Add .ZIP Library...`
+
+2. You will be prompted to select the library you want to add. Navigate to the .zip file's location and open it.
+
+3. The library will automatically be installed.
```

Comment:
I noticed that the libraries doesn't refresh automatically on 2.x, and that this has already been reported.

https://github.com/arduino/arduino-ide/issues/659

Need to confirm how it looks in Zendesk but referencing GitHub issues this way would allow for tracking automatically when they become resolved so we can update the text.

```suggestion
3. If you're using Arduino IDE 2.x, you may need to restart it for the library to be available[^1].

[^1]: https://github.com/arduino/arduino-ide/issues/659
```

But as long as it's mentioned this LGTM 👍

Reply:
Good catch, thanks